src/renderer/src/components/VideoPlayer/controls/PlaybackRateSelector.tsx

```
@@ -34,28 +61,261 @@ export function PlaybackRateSelector({
+  const currentOption = SPEED_OPTIONS.find((option) => option.value === playbackRate)
+
+  // 点击外部关闭下拉框
+  useEffect((): (() => void) | void => {
+    const handleClickOutside = (event: MouseEvent): void => {
+      const target = event.target as Element
+      if (!target.closest('[data-playback-rate-selector]')) {
+        setIsDropdownOpen(false)
+      }
+    }
+
+    if (isDropdownOpen) {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)